.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -86,23 +97,34 @@ This is a checklist for PR authors & reviewers. Please make sure to complete all
     - [ ] Android / Chrome
     - [ ] MacOS / Chrome
     - [ ] MacOS / Desktop
-- [ ] I verified there are no console errors related to changes in this PR
+- [ ] I verified there are no console errors (if there’s a console error not related to the PR, report it or open an issue for it to be fixed)
 - [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
-    - [ ] I verified comments were added when the code was not self explanatory
-    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files (if applicable)
-    - [ ] I verified proper naming convention for platform-specific files was followed (if applicable)
-    - [ ] I verified [style guidelines](https://github.com/Expensify/App/blob/main/STYLING.md) were followed
+    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. “toggleReport” and not “onIconClick”).
+    - [ ] I verified that comments were added to code that is not self explanatory
+    - [ ] I verified that any new or modified comments were clear, correct english, and explained “why” the code was doing something instead of only explaining “what” the code was doing.
+    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files
+    - [ ] I verified any copy / text that was added to the app is correct english and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
+    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named “index.js”. All platform-specific files are named for the platform the code supports as outlined in the README.
     - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) were followed
+- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
 - [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
 - [ ] I verified other components are not impacted by changes in this PR (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
+- [ ] I verified all code is DRY
```

Comment:
While DRYness is good, I think it can be a trade off decision sometimes between not repeating and making the code simple. Being too dry can sometimes result in very abstract code. How can someone say that it has been "verified" for in terms of DRY?